contracts/extension/src/lib.rs

```diff
@@ -0,0 +1,95 @@
+#![cfg_attr(not(feature = "std"), no_std)]
+
+use ink_lang as ink;
+use ink_env::{Environment, DefaultEnvironment, AccountId};
+
+#[ink::chain_extension]
+pub trait SeraiExtension {
+  type ErrorCode = ();
+
+  /// Returns the amount of active validators on the current chain.
+  #[ink(extension = 0, handle_status = false, returns_result = false)]
+  fn active_validators_len() -> u16;
+
+  /// Returns the ID for the current validator set for the current chain.
+  // TODO: Decide if this should be an increasing unsigned integer instead of a hash.
+  #[ink(extension = 1, handle_status = false, returns_result = false)]
+  fn validator_set_id() -> [u8; 32];
```

Comment:
idk if it matters here, but validator set ID in substrate is an incrementing uint

Reply:
It does. Thanks :)

Reply:
Properly defined in the new Validator Sets document (or at least, close enough, as it's defined as a u32 in that doc).